images/tools/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN INSTALL_PKGS="\
   xfsprogs \
   fio \
   stress-ng \
+  python36 \
```

Comment:
Why strictly python 3.6?

Reply:
That was the package available when I tested it locally. I am not partial to a specific minor of 3.x if you have another suggestion.

Reply:
first version of this PR was `python3`, I am wondering is there any reason to use `python36`. If not, I'd prefer to move forward with `python3`?

Reply:
Sounds good, updated

Reply:
@jcpowermac you are right. `python3` gaves an error `package python3 is not installed` with an exit code `1`. We have to use `python36`

Reply:
This was the reason why I set it to python36

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_oc/1682/pull-ci-openshift-oc-master-images/1757077012503072768#1:build-log.txt%3A982